pyseir/ensembles/ensemble_runner.py

```diff
@@ -1,24 +1,28 @@
 import datetime
-import logging
 import os
+from dataclasses import dataclass
+from typing import Mapping, Any, Optional, Iterable
+
 import numpy as np
 from multiprocessing import Pool
 from functools import partial
-import us
+
+import structlog
 import pickle
 import json
 import copy
 from collections import defaultdict
+
+from libs import pipeline
+from pyseir.models import seir_model
 from pyseir.models.seir_model import SEIRModel
 from pyseir.parameters.parameter_ensemble_generator import ParameterEnsembleGenerator
 import pyseir.models.suppression_policies as sp
 from pyseir.utils import get_run_artifact_path, RunArtifact, RunMode
-from pyseir.inference import fit_results
-from libs.datasets import AggregationLevel
 from libs.datasets import combined_datasets
 
 
-_logger = logging.getLogger(__name__)
+_log = structlog.get_logger()
```

Comment:
would prefer to keep the `__name__` so that the logger is not the root logger? (i believe that's the case, though could definitely be wrong)

Reply:
I think this isn't needed because we're using `structlog.stdlib.LoggerFactory()` (via common_init.py) that takes care of getting a name from the stack.